doc-content/enterprise-only/project-data/preview-editor-enable-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='preview-editor-enable-proc']
+= Enabling Test Scenario (Preview) editor
+
```

Comment:
Suggested rephrase:
The *Test Scenario (Preview)* editor is available as a _Technology Preview_ feature in {PRODUCT} and is disabled by default.